database/repositories/beverages.php

```diff
@@ -22,4 +22,26 @@ function get_all_beverages(): array
     return array_column($result->fetch_all(), 0);
 }
 
+function get_users_beverage(string $user): string|null
```

Comment:
Can you rename the parameter to $user_id, and potentially the function name to get_user_beverage_by_id